modules/gcp-interconnect.adoc

```diff
@@ -0,0 +1,40 @@
+:_module-type: PROCEDURE
+//Specify the module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/gcp-private-connections.adoc
+
+[id="template-module_{context}"]
+= Configuring GCP Cloud Interconnect
+
+[role="_abstract"]
+{GCP} Cloud Interconnect provides low latency, high availability connections that enable you to reliably transfer data between your on-premises and Google Cloud Virtual Private Cloud (VPC) networks. Also, Interconnect connections provide internal IP address communication, which means internal IP addresses are directly accessible from both networks.
+
+Cloud Interconnect offers two options for extending your on-premises network:
+
+* *Dedicated Interconnect*: Provides a direct physical connection between your on-premises network and Google's network.
+* *Partner Interconnect*: Provides connectivity between your on-premises and VPC networks through a supported service provider.
+
+.Prerequisites
+
+* The proper link:https://cloud.google.com/network-connectivity/docs/interconnect/how-to/dedicated/ordering-dedicated-interconnect#expandable-1[permissions or IAM roles].
+
```

Comment:
For Partner Interconnect, connection with a supported service provider is required. So I would suggest to add it to prerequisites:
If you want to use Partner Interconnect, you must already have connectivity with a supported service provider.